library/library.go

```diff
@@ -42,7 +44,7 @@ import (
 //   - (*C.char): A pointer to an error message (if any).
 
 //export render_app
-func render_app(pathPtr *C.char, configPtr *C.char, width, height, magnify, maxDuration, timeout, imageFormat, silenceOutput C.int) (*C.uchar, C.int, *C.char, *C.char) {
+func render_app(pathPtr *C.char, configPtr *C.char, width, height, magnify, maxDuration, timeout, imageFormat, silenceOutput C.int, filtersPtr *C.char) (*C.uchar, C.int, *C.char, *C.char) {
```

Comment:
This changes the API contract, so be sure to change the caller site in tronbyt/server when bumping the version here.

Reply:
Yep, I saw https://github.com/tronbyt/server/pull/221 and figured we have some flexibility.
Other option is to push this into the `config` JSON string and pop it out pixlet side...

Reply:
I think this is the cleaner solution than merging this into the app config. Just wanted to make sure you know about the workflow:
1. merge this
2. bump the pixlet version and create a new release
3. pick up the new pixlet on the server side and change the function signature there at the same time